source/multigrid/mg_transfer_matrix_free.cc

```diff
@@ -373,83 +387,90 @@ ::do_prolongate_add (const unsigned int                           to_level,
       const unsigned int n_chunks = cell+vec_size > n_owned_level_cells[to_level-1] ?
                                     n_owned_level_cells[to_level-1] - cell : vec_size;
 
-      // read from source vector
-      for (unsigned int v=0; v<n_chunks; ++v)
+      // FIXME: for starters, try a simple loop over blocks. In reality,
+      // one has to do work on several blocks of vectors at a time.
+      for (unsigned int b = 0; b < internal::get_n_blocks(dst); ++b)
```

Comment:
@kronbichler this was my concern about a naïve outer loop over blocks.

@masterleinad : since you plan to add transfer where each block has a different DoFHandler attached, how do you think we should structure things so that we re-use as much as possible the existing single block code?